src/Common/ruleset.xml

```diff
@@ -18,6 +18,13 @@
         <priority>1</priority>
     </rule>
 
+    <rule
+            name="ImplementsApiInheritDocRule"
+            message="Access: {0}"
+            class="ArchitectureSniffer\Common\ImplementsApiInheritDocRule">
+
+        <priority>3</priority>
```

Comment:
@dereuromark I've 1 issue here. Even with prio 3, it still causes errors. What to do here? Should I fix entire Spryker/spryker? :S